client/my-sites/plan-features-2023-grid/style.scss

```diff
@@ -307,6 +315,7 @@
 		border: none;
 		background-color: var(--color-surface);
 		position: relative;
+		vertical-align: baseline;
```

Comment:
Hey @claudiucelfilip this results in a broken view Automattic/martech#1496
![image](https://user-images.githubusercontent.com/3422709/218315486-e2f046b7-92a3-4be7-9783-662a501eb712.png)

Can it be removed? I am trying to figure out what change was it was used for. Without this line, the link-in-bio flow looks like this.


<img width="1647" alt="image" src="https://user-images.githubusercontent.com/3422709/218318359-d02a0ff3-a9c9-404d-a8da-7d0398da7db5.png">


Side Note, we need to probably remove the free plan text below since we are showing the free plan here.
`Unlock a powerful bundle of features for your Link in Bio. Or start with a free plan.`

Reply:
> Can it be removed? I am trying to figure out what change it was used for. Without this line, the link-in-bio flow looks like this. Nothing seems to be broken here.

@jdc91 Removing it causes the cells to be misaligned (on dev environment at least):

![Annotation on 2023-02-13 at 10-13-15](https://user-images.githubusercontent.com/2749938/218404838-db7b396b-baeb-4fce-b671-5c37859bf533.png)

In the other flows, the alignment it's already been set by another selector but it doesn't seem to be shared in this flow.
I think the problem is that the button cells need to be set to `bottom` and the selector for doing this (`.plan-features-2023-grid__table-item.is-top-buttons`) doesn't have enough specificity:

![Annotation on 2023-02-13 at 10-20-15](https://user-images.githubusercontent.com/2749938/218406135-565c4012-2c88-4154-aac6-99144c049c2d.png)

Adding another class seems to fix it:

![Annotation on 2023-02-13 at 10-18-56](https://user-images.githubusercontent.com/2749938/218406209-09efd521-b6ce-4d54-ba74-b5c3b6861503.png)